app/src/etc/supervisord.conf

```diff
@@ -46,7 +46,17 @@ command = node NodeJSServer.js
 directory=/home/bdt/map/src/map_nodejs_server/
 stdout_logfile=/var/log/NodeJSServer.log
 stderr_logfile=/var/log/NodeJSServer.log
-autostart=false                ; start at supervisord start (default: true)
+autostart=true                 ; start at supervisord start (default: true)
 autorestart=false              ; restart at unexpected quit (default: true)
 stopasgroup=true               ; stop all subprocesses together
 killasgroup=true               ; kill all subprocesses together
+
+[program:startup]
+command = /bin/bash /home/bdt/map/src/map_scripts/startup_after_shutdown.sh
```

Comment:
one way would be to configure `/etc/supervisor.d/*.conf` and then use `docker run -v <host_dir>:/etc/supervisor.d ...`.
drop your scripts and configs into that mount and you're good to go.

alternatively (and my preference for many things): create a docker image `FROM scratch` including those files with `VOLUME /etc/supervisord.d`, then `docker create --name private_files <file_image> true` followed by `docker run --volumes-from private_files ...`